streaming/src/main/scala/org/apache/spark/streaming/scheduler/ReceiverTracker.scala

```diff
@@ -227,6 +255,8 @@ class ReceiverTracker(ssc: StreamingContext, skipReceiverLaunch: Boolean = false
       case DeregisterReceiver(streamId, message, error) =>
         deregisterReceiver(streamId, message, error)
         context.reply(true)
+      case ScheduleReceiver(streamId) =>
```

Comment:
Or if we want to maintain the naming scheme (which uses `Register` instead `Start`), it may be better to name the new one `ValidateLocation(streamId, host)`